packages/amplify-provider-awscloudformation/src/__tests__/utils/admin-login-server.test.ts

```diff
@@ -28,4 +29,17 @@ describe('AdminLoginServer', () => {
     expect(postMock).toBeCalled();
     expect(listenMock).toBeCalledWith(4242, '0.0.0.0');
   });
+  
+  test('shut down running server', async () => {
+    const adminLoginServer = new AdminLoginServer('appId', 'http://example.com', context_stub.print);
+    listenMock.mockReturnValue({ close: serverCloseMock });
+
+    await new Promise<void>(resolve => {
+      adminLoginServer.startServer(() => {
+      });
+      resolve();
```

Comment:
```suggestion
      adminLoginServer.startServer(resolve);
```

Reply:
Thanks

Reply:
Hey @cjihrig, when I make these changes, a Jest timeout error occurs.

````
Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Error
````

Also, I have taken the code to start the `adminLoginServer` from [this test](https://github.com/aws-amplify/amplify-cli/blob/07525b3b4361f5af673687bc265e97ceba359a13/packages/amplify-provider-awscloudformation/src/__tests__/utils/admin-login-server.test.ts#L24).

Reply:
The mocking must change the behavior so that the `startServer()` callback is never actually called? I guess it's fine for testing purposes, but it's not ideal 😄

Reply:
Yep, I think that is the reason 👍🏻